q2_feature_classifier/tests/test_cutter.py

```diff
@@ -98,7 +98,7 @@ def test_extract_reads_fail_max_length(self):
                 max_length=1)
 
     def test_extract_reads_fail_trim_entire_read(self):
```

Comment:
good catch that this test will trigger this error. However, it looks like the purpose of this test was to error out if all read lengths == 0.

Please split these into two tests. `test_extract_reads_fail_trim_entire_read` should use `min_length=0` so that we are testing for the original error message. Please add a new test for `min_length>trunc_len`